modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/base/ListTypeEntryLocalServiceBaseImpl.java

```diff
@@ -133,7 +133,8 @@ public ListTypeEntry deleteListTypeEntry(long listTypeEntryId)
 	 */
 	@Indexable(type = IndexableType.DELETE)
 	@Override
-	public ListTypeEntry deleteListTypeEntry(ListTypeEntry listTypeEntry) {
+	public ListTypeEntry deleteListTypeEntry(ListTypeEntry listTypeEntry)
+		throws PortalException {
```

Comment:
Can you separate code changed manually and code changed by buildService into different commits, please?